src/components/TableHeader/index.js

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components'
+import { colors } from '../../styles'
+
+export const TableHeader = styled.div`
+    display: flex;
+    flex-direction: row;
```

Comment:
`flex-direction: row` is the default, so unless this is an override you probably don't need this.

Reply:
yes true